index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width">
+        <meta name="keywords" content="web design, frontend web development, software engineer, javascript engineer">
+        <title>Raphael Donanu | My Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.7.2/animate.min.css">
+    </head>
+    <body>
+        <nav class="zone1">
```

Comment:
Why not add a `header` tag before the `nav` tag? Also, you may want to use a more descriptive class name like `nav-bar` instead of `zone1`

Reply:
Thank you for pointing this out. I have modified it.